cmd/configure.go

```diff
@@ -125,6 +151,7 @@ func configureCmdFunc(cmd *cobra.Command, args []string) {
 	})
 
 	c, _ := yaml.Marshal(config.GlobalConfig)
+	c = append(c, defaultProviderConfig...)
```

Comment:
Not sure about this. What do you think about adding it as commented block?

Reply:
It super confusing for new users that goship doesn't work after `configure` (been there two times and observed teammates). As it supports EC2 only for now, I would leave it here. Notice at the end of `configure` encourages user to check sample provider config and tune it.